Sources/ComposableArchitecture/Store.swift

```diff
@@ -209,4 +209,6 @@ public struct StoreProducer<State>: SignalProducerConvertible {
   where LocalState: Equatable {
     self.producer.map(keyPath).skipRepeats()
   }
+    
+  public var state: Effect<State, Never> { producer }
```

Comment:
Can you remove this and please? I need to think some more about how to address this problem.